tools/fork-network/src/cli.rs

```diff
@@ -416,15 +416,33 @@ impl ForkNetworkCommand {
         let storage = open_storage(&home_dir, near_config).unwrap();
         let store = storage.get_hot_store();
 
-        tracing::info!("Delete unneeded columns in the original DB");
-        let mut update = store.store_update();
-        for col in DBCol::iter() {
-            match col {
-                DBCol::DbVersion | DBCol::Misc | DBCol::State | DBCol::FlatState => {}
-                _ => update.delete_all(col),
+        let store_path = home_dir
+            .join(near_config.config.store.path.clone().unwrap_or_else(|| PathBuf::from("data")));
+        let temp_store_home = store_path.join("temp-trimmed-db");
+        if std::fs::metadata(&temp_store_home).is_ok() {
+            panic!("Temp trimmed DB already exists; please delete temp-trimmed-db in the data directory first");
```

Comment:
why not return an anyhow::Error instead of panicking?

Reply:
Oh, sure.